pkg/loki/modules.go

```diff
@@ -178,6 +178,55 @@ func (t *Loki) stopStore() error {
 	return nil
 }
 
+// listDeps recursively gets a list of dependencies for a passed moduleName
+func listDeps(m moduleName) []moduleName {
+	deps := modules[m].deps
+	for _, d := range modules[m].deps {
+		deps = append(deps, listDeps(d)...)
+	}
+	return deps
+}
+
+// getDeps gets a list of all dependencies ordered so that items are always after any of their dependencies.
+func getDeps(m moduleName) []moduleName {
+	deps := listDeps(m)
+
+	// get a unique list of moduleNames, with a flag for whether they have been added to our result
+	uniq := make(map[moduleName]bool)
```

Comment:
Throughout the Loki codebase I've preferred `map[moduleName]bool{}` over make, unless we know the size.  Super nit picky, sorry.